tests/functional/base.py

```diff
@@ -17,8 +20,105 @@
 from custodia.server.config import parse_config
 
 
+def wait_pid(process, wait):
```

Comment:
Why did you move the functions out of the class? If the test class doesn't suite your needs, please talk to @raildo and modify the base class.